docs/validate-stable-rc-builds.md

```diff
@@ -0,0 +1,237 @@
+# Stable rc review validation process
+Following steps to be performed when a stable rc review pushed on stable mailing
+list
+
+## Wait for stable rc review email
+Greg KH pushes on to stable rc trees and creates a tag
+then sends out email on stable ML with the list of patches included in that
+branch.
+
+stable-rc tree link:
+
+*https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable-rc.git*
+
+stable-rc review email example link,
+
+*https://www.spinics.net/lists/stable/msg320559.html*
```

Comment:
That's a stable release. Maybe we can use this RC review request from today: https://lore.kernel.org/stable/20190808190453.582417307@linuxfoundation.org/T/#mda3573d436265842df547701dc61f9e236627d6c

Reply:
yeah this one looks good.
I have updated this link in the document.